systemtest/src/test/java/io/strimzi/systemtest/multiplenamespace/AllNamespaceST.java

```diff
@@ -2,14 +2,15 @@
  * Copyright Strimzi authors.
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
-package io.strimzi.systemtest;
+package io.strimzi.systemtest.multiplenamespace;
```

Comment:
I wonder if all the namespace tests should go in a package like `watcher`
i.e. io.strimzi.systemtest.watcher.AllNamespaceST
io.strimzi.systemtest.watcher.MultipleNamespaceST

The package being the intent/theme of all the tests and then the class being more specific